### DISCUSSION TITLE
missing Tags

### DISCUSSION THREAD

**Discussion by swschilke** [dailydotdev/daily#1313] [General]
Hi, I would like to have a tag für nlu, Graphrag (didn't check ;-)), Podcast etc.
ThanX

**Comment by idoshamun** [comment]
Hi @swschilke,

I'm not aware of nlu and graphrag. Can you elaborate please?

**Comment by swschilke** [comment]
NLU is natural language understanding, a step further from natural language
processing (nlp)

GraphRAG is also a step further from RAG, just with a knowledge graph for
representing the (private) information

Hope this explains it

Podcast, would be for podcasts or podcast episodes which are published in
the genre or domain

**Reply by idoshamun** [reply]
We currently have NLP and RAG, not the specific ones you mentioned. Once we have enough posts about NLU and GraphRAG you can expect to have them as tags :)

**Comment by swschilke** [comment]
Maybe the poeple tag it NLP or RAG as there is no Tags for NLU or GraphRag
- if they are there the posts will come ;-)

**Reply by nimrodkra** [reply]
Our system automatically identifies and selects the tags. It doesn't rely on human input. The auto-tagging system looks into the content itself and what keywords are in it, and based on that, it determines if there's "enough content" for a certain tag. If more people wrote content (or published videos) about NLU and GraphRAG out there on the internet, the likelihood for these tags to be added to daily.dev would increase. In other words, the bottleneck is not the tagging system, it's the shortage of such content online. 

If you know any publications that specialize in this niche, feel free to suggest them here, and we will look into it (just make sure it complies with the [content guidelines](https://docs.daily.dev/docs/for-content-creators/content-guidelines) first)

**Comment by swschilke** [comment]
So some links like this:

https://arxiv.org/abs/2404.16130

https://github.com/microsoft/graphrag

https://www.microsoft.com/en-us/research/project/graphrag/

https://youtu.be/jCjyaQL-7mA

https://www.ontotext.com/knowledgehub/fundamentals/what-is-graph-rag/

https://www.microsoft.com/en-us/research/blog/graphrag-unlocking-llm-discovery-on-narrative-private-data/

https://neo4j.com/blog/graphrag-manifesto/

https://docs.llamaindex.ai/en/stable/examples/query_engine/knowledge_graph_rag_query_engine/

https://neo4j.com/developer-blog/graphrag-ecosystem-tools/

https://jeongiitae.medium.com/from-rag-to-graphrag-what-is-the-graphrag-and-why-i-use-it-f75a7852c10c

https://bhavikjikadara.medium.com/graphrag-advanced-data-retrieval-for-enhanced-insights-bcef777404d2
